boards/hail/src/main.rs

```diff
@@ -35,15 +35,15 @@ unsafe fn load_processes() -> &'static mut [Option<kernel::process::Process<'sta
         static _sapps: u8;
     }
 
-    const NUM_PROCS: usize = 2;
+    const NUM_PROCS: usize = 4;
```

Comment:
Side-note: We can totally do this, the problem is figuring out how to address processes from the rest of the system (e.g. when we enter a grant). We need some sort of collection, an array is clearly the fastest to access, a linked list would be simple to implement if the memory was taxes to the app itself but would be much slower to access, a tree maybe somewhere in the middle?

On the flip side, we could keep it as an array, but given that a the array would be storing only pointers (rather than a whole process struct) we could probably get away with just overdoing the max capacity without much memory overhead.